### ISSUE TITLE
Notification pattern issues

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1259]
Various issues found with the notifications dropdown pattern during accessibility audit:

1. CSS bug with dismiss buttons, buttons are fixed position when scrolling (see screenshot)
2. Markup isn't as semantic as it could be
3. Contrast failures on time ago text (muted) and View release notes link in new notification

![Screenshot 2020-06-11 at 12 47 57](https://user-images.githubusercontent.com/756393/84504821-81cce100-acb4-11ea-8711-9e58dd48bb9f.png)
